<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VstWizard</class>
 <widget class="QWizard" name="VstWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="HackWidget" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Step 3. &lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#000000;&quot;&gt;Choose a VSTi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tip: Choose &quot;Manage VST instruments&quot; under the &quot;Creator Live&quot; menu to add and remove VST instruments.</string>
      </property>
      <property name="buddy">
       <cstring>listWidget_vsti</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_vsti"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Step 4. &lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#000000;&quot;&gt;Choose an instance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tip: an instance is a VSTi window. You can use the same VSTi window in more than one track by using the same instance.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_inst"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Step 5. &lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#000000;&quot;&gt;Choose a channel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tip: A channel is a MIDI input and a stereo audio output. Channel 'A' is the first MIDI output and the first two audio outputs (i.e., the first stereo audio output), and so on... Some VSTis only support one channel, others such as Kontakt support many.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_chan"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Step 6. &lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#000000;&quot;&gt;Choose an output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Tip: You can manage your audio outputs by selecting &quot;Audio Setup...&quot; from the &quot;Creator Live&quot; menu.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_out"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HackWidget</class>
   <extends>QWizardPage</extends>
   <header>hackwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
